packages/trader/src/Modules/SmartChart/Components/__tests__/control-widgets.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import ControlWidgets from '../control-widgets';
+
+jest.mock('Modules/SmartChart', () => ({
+    ...jest.requireActual('Modules/SmartChart'),
+    ChartMode: () => <div>MockedChartMode</div>,
+    DrawTools: () => <div>MockedDrawTools</div>,
+    Share: () => <div>MockedShare</div>,
+    StudyLegend: () => <div>MockedStudyLegend</div>,
+    Views: () => <div>MockedViews</div>,
+}));
+
+describe('<ControlWidgets />', () => {
+    const store_config = mockStore({});
+    const renderComponent = ({ mocked_store_props = store_config }) =>
+        render(
+            <StoreProvider store={mocked_store_props}>
+                <ControlWidgets />
+            </StoreProvider>
+        );
+    const mocked_store_props = mockStore({
+        contract_trade: {
+            updateChartType: jest.fn(),
```

Comment:
These functions already exist in _packages/stores/src/mockStore.ts_.

Reply:
sure but how to use?

Reply:
If you are not going to check if it was called or not, then it can be omitted. Try just `mockStore({})`. If something is already mocled and you need default value, then it should work